packages/node/src/transports/http.ts

```diff
@@ -46,8 +46,8 @@ export class HTTPTransport implements Transport {
   /**
    * @inheritDoc
    */
-  public async sendPayload(payload: Payload): Promise<Response> {
-    const call = async (): Promise<Response> => await this._sendWithModule(payload, REQUEST_CANCEL_TIMEOUT);
+  public async sendPayload(payload: Payload, limitInMs = REQUEST_CANCEL_TIMEOUT): Promise<Response> {
```

Comment:
Tweaked the signature of `sendPayload` to accept a timeout miliseconds number and pass that down to _sendWithModule. This makes it possible to test with nock,  plus there's a ticket out to make the timeout period an optional parameter to the client, and this helps pave the way for that change.